### ISSUE TITLE
Clearing and changing folder-search does not update folder-list on top

### ISSUE THREAD

**Issue by neufeind** [opto/nostalgy-xpi#61]
If I type "something" I get a folderlist with matches on top of the searchbox. If I now mark everything in the box (in my case: with Shift-Pos1) and enter "else" I still get the previous results in the list on top. But using up/down-cursorkeys I get correct matches inside the searchbox itself, just the list on top does not seem to be updated anymore.

**Comment by opto** [comment]
not sure what you mean, can you revise the example. Which TB/N++/OS?

On Win10, shift-pos1 kills off the box with matches and looses the N++ functionality.
But I don't quite understand what you want to achieve.

Klaus

**Comment by neufeind** [comment]
On Linux (KDE) I use Shift-Pos1 to mark the text and backspace or Del to empty the field ... or simply type another letter after marking everything.
While typing I get an autocomplete, first need to move to "End" and then "Shift-Pos1" so even the automatic completion-text after it is correctly removed. But that is doable.

If I remember correctly at the time when I filed this issue even after marking everything and typing another letter the list would not refresh (but stay with the old content). That works now / was fixed on a later release or whatever.

So I'm closing this issue, since it "works for me" (meanwhile).
